ProcessLib/LIE/SmallDeformation/CreateSmallDeformationProcess.cpp

```diff
@@ -98,9 +98,14 @@ std::unique_ptr<Process> createSmallDeformationProcess(
         process_variables;
     process_variables.push_back(std::move(per_process_variables));
 
+    auto const material_ids =
```

Comment:
Can it be a member of mesh class? It appears in several functions.

Reply:
I mean to add a member to Mesh as
```
PropertyVector<int> Mesh::getMaterialIDs() const
{
return _properties.existsPropertyVector<int>("MaterialIDs")
            ? _properties.getPropertyVector<int>("MaterialIDs")
            : nullptr;
}
```

Reply:
What do other think if we add a free function in MeshLib namespace?
```c++
PropertyVector<int>* materialIDs(Mesh const&);
```

Reply:
Sounds good.